dbt_project.yml

```diff
@@ -64,6 +64,11 @@ models:
       tests:
         fct_missing_primary_key_tests:
           +enabled: false
+        fct_sources_without_freshness:
+          +enabled: false
+      documentation:
+        fct_undocumented_sources:
+          +enabled: false
```

Comment:
just curious: are these tests that you would like us to use in the future? (eventually enabled = true) 

or were you getting an error in CI?

Reply:
dbt_project_evaluator runs all these checks by default and therefore the CI was failing. We could turn it on later on eventually if we decide to follow best practices by dbt.

Otherwise, we could leave it as is

Reply:
ah, ok. In that case leave it as is and we can revisit